src/aioftp/client.py

```diff
@@ -619,7 +758,8 @@ class Client(BaseClient):
     :param **siosocks_asyncio_kwargs: siosocks key-word only arguments
     """
 
-    async def connect(self, host, port=DEFAULT_PORT):
+    @override
+    async def connect(self, host: str, port: int = DEFAULT_PORT) -> Union[Code, List[str]]:
```

Comment:
Why this is a `Union` if the only return type is `list[str]`?

Reply:
Fixed